### ISSUE TITLE
[Arcs] Event card should give pips if Imperial Council is cancelled

### ISSUE THREAD

**Issue by turmoilbyrd** [LederCards/cards#109]
Now that Imperial Council can be secured then cancelled, should Event card be able to function again once Council is cancelled and returned to the Court?

Current rule:
> [Limitations.](https://arcs.seiyria.com/#16.1.2.1-limitations)Your Event card gives you no actions if another Event card was played earlier this round or if the Imperial Council was secured earlier this round (see [16.3](https://arcs.seiyria.com/#16.3-imperial-council)).

The easiest way is perhaps to say

> ...or if the Imperial Council was <s>secured earlier this round</s> tucked under a card.

related
-  [Council Intrigue](https://cards.ledergames.com/card/ARCS-CC12)
- [Council Insiders](https://cards.ledergames.com/card/ARCS-F116))

**Comment by turmoilbyrd** [comment]
This is how I'd intuit it (will be weird to say the Council was cancelled but Event cards still don't give pips), but it may be a slightly tricky to write? Because in https://arcs.seiyria.com/#16.3.3-securing-the-council , "tucking" is just "as a reminder", implying it's optional..? Practically I think everyone do it anyway though.

**Comment by jdyearsley** [comment]
Yes, I think that's sensible. Could you also edit to link to the original "secured then cancelled" discussion for posterity?

**Comment by turmoilbyrd** [comment]
Honestly I don't even remember when we had that conversation, but it resulted in the FAQ on [Council Intrigue](https://cards.ledergames.com/card/ARCS-CC12)

**Comment by yggdrasilsYeoman** [comment]
Howdy Josh, here are the previous discussions I believe you were looking for!

https://discord.com/channels/476234833572397056/963112189802201108/1318532368100753489

https://discord.com/channels/476234833572397056/963112189802201108/1386412891468464322

**Comment by jdyearsley** [comment]
Rulebook edited, and FAQ is slated to be added.
